bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/storage/directentrylogger/TestDirectEntryLogger.java

```diff
@@ -200,7 +200,10 @@ public void close() throws IOException {
             }
             elog.flush();
             for (Long loc : locations) {
-                ReferenceCountUtil.release(elog.readEntry(loc));
+                int i = locations.indexOf(loc);
```

Comment:
This test is for direct entry logger, it can't cover this fix

Reply:
@horizonzy  Fixed Thx for pointing it out